cypress/e2e/capture.cy.ts

```diff
@@ -287,7 +287,7 @@ describe('Event capture', () => {
 
                 // the code below is going to trigger an event capture
                 // we want to assert on the request
-                cy.intercept('POST', '**/e/*', async (request) => {
```

Comment:
as far as I could tell from testing

`**/blah` matches `something/blah` but not `/blah` so wasn't matching all requests the tests were making